frontend/path_to_case_component

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useQuery } from '@apollo/client';
+import { CASE_DETAILS_QUERY } from './queries';
+
+const CaseDetailsComponent = ({ caseId }) => {
+  const { data, loading, error } = useQuery(CASE_DETAILS_QUERY, {
+    variables: { id: caseId },
+  });
+
+  if (loading) return <p>Loading...</p>;
+  if (error) return <p>Error: {error.message}</p>;
+
+  const { name, status, priority, code } = data.case;
```

Comment:
Consider handling the case where the `data.case` is `null` or `undefined` to prevent potential runtime errors. [medium]